snap7/client.py

```diff
@@ -534,3 +534,15 @@ def get_param(self, number):
                                          byref(value))
         check_error(code)
         return value.value
+
+    def get_pdu_length(self):
```

Comment:
why not add requested and negotiated as arguments to the `get_pdu_length()` function?